capa/main.py

```diff
@@ -1123,8 +1124,12 @@ def main(argv=None):
             if not (args.verbose or args.vverbose or args.json):
                 logger.debug("file limitation short circuit, won't analyze fully.")
                 return E_FILE_LIMITATION
-
-    if format_ == FORMAT_FREEZE:
+    if format_ == FORMAT_RESULT:
+        with open(args.sample, "rb") as f:
+            buf = f.read()
+        buf.decode("utf-8")    
+        meta, capabilities = capa.render.result_document.ResultDocument.parse_raw(buf,rules)
+    elif format_ == FORMAT_FREEZE:
         with open(args.sample, "rb") as f:
             extractor = capa.features.freeze.load(f.read())
```

Comment:
i think the logic here is broken because the `meta, capabilities` aren't extracted.

Reply:
i assume our test cases should have caught this, too. have you run the test suite with the changes proposed so far?